bindata/v3.11.0/config/route-controller-defaultconfig.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: openshiftcontrolplane.config.openshift.io/v1
```

Comment:
route-controller-defaultconfig.yaml -> route-controller-manager-defaultconfig.yaml

Reply:
I have changed all route-controller-manager files to have a consistent prefix